Libplanet/Net/DifferentAppProtocolVersionException.cs

```diff
@@ -23,10 +23,13 @@ public sealed class DifferentAppProtocolVersionException : Exception
         /// </param>
         public DifferentAppProtocolVersionException(
             string message,
-            int expectedVersion,
-            int actualVersion)
-            : base($"Expected Version: {expectedVersion}, " +
-                   $"Actual Version: {actualVersion}, {message}")
+            AppProtocolVersion expectedVersion,
+            AppProtocolVersion actualVersion)
+            : base($"{message}\n" +
+                   $"Expected Version: {expectedVersion} ({expectedVersion.Extra}) " +
+                   $"[{ByteUtil.Hex(expectedVersion.Signature)} by {expectedVersion.Signer}]\n" +
+                   $"Actual Version: {actualVersion} ({actualVersion.Extra}) " +
```

Comment:
```suggestion
                   $"Expected Version: {expectedVersion} " +
                   $"[{ByteUtil.Hex(expectedVersion.Signature)} by {expectedVersion.Signer}]\n" +
                   $"Actual Version: {actualVersion} " +
```

If it had `Extra` value, `AppProtocolVersion.ToString()` will return `{Version} {Extra}`.
So it doesn't need to insert `Extra` field one more, does it?

Reply:
You're right. I'll fix this too.